src/test/java/tests/ShoppingCartTest.java

```diff
@@ -0,0 +1,58 @@
+package tests;
+
+import com.epam.tamentoring.bo.Product;
+import com.epam.tamentoring.bo.ShoppingCart;
+import com.epam.tamentoring.exceptions.ProductNotFoundException;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class ShoppingCartTest {
+
+    @Test
+    public void testAddProductsToShoppingCart() {
+        ShoppingCart shoppingCart = new ShoppingCart(new ArrayList<>());
+        Product product = new Product();
+        shoppingCart.addProductToCart(product);
+        assertThat(shoppingCart.getProducts()).contains(product);
```

Comment:
I would recommend adding the second check on the size of the cart

Reply:
Added check that the shopping card is empty after the only product removal